docs/11.EXT-REFERENCE-MODULE.md

```diff
@@ -0,0 +1,116 @@
+#  Modules API
+
+## jerryx_module_resolve
+
+**Summary**
+
+Load a copy of a module into the current context or return one that was already loaded if it is found.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerryx_module_resolve (const jerry_char_t *name);
```

Comment:
The convention for extension seems to be `jerryx_<extension>_what_the_api_does ()`. Also, `resolve` is used in the spec to refer to loading modules.